modyn/storage/include/internal/grpc/storage_service_impl.hpp

```diff
@@ -230,6 +230,8 @@ class StorageServiceImpl final : public modyn::storage::Storage::Service {
     session.close();
 
     if (file_ids.empty()) {
+      SPDLOG_INFO("No files found for dataset {} with start_timestamp = {} and end_timestamp = {}", dataset_id,
+                  start_timestamp, end_timestamp);
```

Comment:
when no data is found, e.g. due to bug in CSVFileWrapper for corrupted lines, this log is very useful.